src/main/java/emissary/analyze/Analyzer.java

```diff
@@ -1,19 +1,25 @@
 package emissary.analyze;
 
 import emissary.core.IBaseDataObject;
+import emissary.core.IBaseDataObjectHelper;
 import emissary.place.ServiceProviderPlace;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.List;
-import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 /**
  * Base class for analyzers
  */
 public abstract class Analyzer extends ServiceProviderPlace {
-    protected List<String> PREFERRED_VIEWS;
-    protected String ANALYZED_DATA_NAME;
+    protected List<String> preferredViews;
+    protected String analyzedDataName;
+    protected List<Pattern> perferredViewPatternList;
```

Comment:
Minor typo.  Might also consider dropping the `List` suffix and just pluralizing to `preferredViewPatterns`. 
```suggestion
    protected List<Pattern> preferredViewPatternList;
```